src/styles.css

```diff
@@ -138,3 +138,52 @@ table {
 body {
   background: #e15c6a;
 }
+
+#searchbar {
```

Comment:
You should add the styling here and take it off of `input` and `select`. You don't want to explicitly style the top-level `input` or `select` elements. 

We might want to talk about this in-person. Its a good opportunity to talk about some big coding ideas.